website/docs/components/dropdown/partials/specifications/states.md

```diff
@@ -0,0 +1,43 @@
+## States
+
+### Button
+
+![Example of dropdown button states](/assets/components/dropdown/dropdown-state-button.png)
+
+### Icon
+
+![Example of dropdown icon states](/assets/components/dropdown/dropdown-state-icon.png)
+
+!!! Info
+
+**A note on disabled states** 
+
+Because disabled states completely remove the interactive function of an element, it can be challenging for a user to understand why it has been disabled and/or why they cannot interact with that element. In an effort to avoid this confusion, we opt for using methods like enabling or hiding the element and, thus, are not offering a disabled state for the Dropdown Toggle. [Read more about when to enable vs hide](https://docs.google.com/document/d/1fqsXjjPnz5HK2NcY1buh5RcI5S6XCgQwfr8GP3kClv0/edit#heading=h.52ub6bvbvcb7)
+
+!!!
+
+<!-- Can't get the mock states to work here for some reason -->
```

Comment:
there's an issue to actually implement these still

Reply:
Should these be moved to the showcase?

Reply:
Oh, good point. Probably.